lib/publiccloud/utils.pm

```diff
@@ -113,6 +113,17 @@ sub registercloudguest {
     record_info('registercloudguest time', 'The command registercloudguest took ' . (time() - $cmd_time) . ' seconds.');
 }
 
+sub register_openstack {
```

Comment:
Shall we move this method to `lib/publiccloud/openstack.pm`?

Reply:
Oh, I haven't thought about that. Nice point. The only thing that pops to my mind is that we might have mixed the purpose of the that library. IMHO, `lib/publiccloud/openstack.pm` is more about provisioning routines than runtime tasks. OTOH, no objects to move it.

Reply:
yes, also good point. Let's keep it as it is :)